### PR TITLE
metrics: Revert "metrics: Fix boot times metrics baseline"

### PR DESCRIPTION
**Pull request by fidencio**
This reverts commit e0029ca049fad3b4b34532714fdb8cf0bc5bd0a4.

Signed-off-by: Fabiano Fidêncio <fidencio@redhat.com>

### CODE REVIEW

**Comment by fidencio** [comment]
```
12:39:02 Report Summary:
12:39:02 +-----+----------------------+-------+--------+--------+-------+--------+--------+---------+-------+-----+
12:39:02 | P/F |         NAME         |  FLR  |  MEAN  |  CEIL  |  GAP  |  MIN   |  MAX   |   RNG   |  COV  | ITS |
12:39:02 +-----+----------------------+-------+--------+--------+-------+--------+--------+---------+-------+-----+
12:39:02 | P   | boot-times           | 95.0% | 96.8%  | 105.0% | 10.0% | -40.8% | 253.7% | -721.7% | 49.4% |  20 |
12:39:02 | P   | memory-footprint     | 95.0% | 101.0% | 105.0% | 10.0% | 101.0% | 101.0% | 0.0%    | 0.0%  |   1 |
12:39:02 | P   | memory-footprint-ksm | 95.0% | 101.1% | 105.0% | 10.0% | 101.1% | 101.1% | 0.0%    | 0.0%  |   1 |
12:39:02 | P   | blogbench            | 90.0% | 101.8% | 110.0% | 20.0% | 101.8% | 101.8% | 0.0%    | 0.0%  |   1 |
12:39:02 | P   | blogbench            | 90.0% | 98.2%  | 110.0% | 20.0% | 98.2%  | 98.2%  | 0.0%    | 0.0%  |   1 |
12:39:02 +-----+----------------------+-------+--------+--------+-------+--------+--------+---------+-------+-----+
```

**Comment by fidencio** [comment]
Let me re-run a few more times to have a feeling of how stable it can be.

**Comment by fidencio** [comment]
```
13:10:29 Report Summary:
13:10:29 +-----+----------------------+-------+--------+--------+-------+--------+--------+----------+-------+-----+
13:10:29 | P/F |         NAME         |  FLR  |  MEAN  |  CEIL  |  GAP  |  MIN   |  MAX   |   RNG    |  COV  | ITS |
13:10:29 +-----+----------------------+-------+--------+--------+-------+--------+--------+----------+-------+-----+
13:10:29 | P   | boot-times           | 95.0% | 101.8% | 105.0% | 10.0% | -8.4%  | 258.7% | -3184.6% | 42.2% |  20 |
13:10:29 | P   | memory-footprint     | 95.0% | 101.0% | 105.0% | 10.0% | 101.0% | 101.0% | 0.0%     | 0.0%  |   1 |
13:10:29 | P   | memory-footprint-ksm | 95.0% | 101.2% | 105.0% | 10.0% | 101.2% | 101.2% | 0.0%     | 0.0%  |   1 |
13:10:29 | P   | blogbench            | 90.0% | 97.0%  | 110.0% | 20.0% | 97.0%  | 97.0%  | 0.0%     | 0.0%  |   1 |
13:10:29 | P   | blogbench            | 90.0% | 97.9%  | 110.0% | 20.0% | 97.9%  | 97.9%  | 0.0%     | 0.0%  |   1 |
13:10:29 +-----+----------------------+-------+--------+--------+-------+--------+--------+----------+-------+-----+
```

**Comment by fidencio** [comment]
```
13:43:49 Report Summary:
13:43:49 +-----+----------------------+-------+--------+--------+-------+--------+--------+---------+-------+-----+
13:43:49 | P/F |         NAME         |  FLR  |  MEAN  |  CEIL  |  GAP  |  MIN   |  MAX   |   RNG   |  COV  | ITS |
13:43:49 +-----+----------------------+-------+--------+--------+-------+--------+--------+---------+-------+-----+
13:43:49 | P   | boot-times           | 95.0% | 102.3% | 105.0% | 10.0% | 13.2%  | 256.0% | 1835.4% | 39.1% |  20 |
13:43:49 | P   | memory-footprint     | 95.0% | 101.0% | 105.0% | 10.0% | 101.0% | 101.0% | 0.0%    | 0.0%  |   1 |
13:43:49 | P   | memory-footprint-ksm | 95.0% | 101.1% | 105.0% | 10.0% | 101.1% | 101.1% | 0.0%    | 0.0%  |   1 |
13:43:49 | P   | blogbench            | 90.0% | 97.6%  | 110.0% | 20.0% | 97.6%  | 97.6%  | 0.0%    | 0.0%  |   1 |
13:43:49 | P   | blogbench            | 90.0% | 98.9%  | 110.0% | 20.0% | 98.9%  | 98.9%  | 0.0%    | 0.0%  |   1 |
13:43:49 +-----+----------------------+-------+--------+--------+-------+--------+--------+---------+-------+-----+
```

**Comment by fidencio** [comment]
```
14:16:28 Report Summary:
14:16:28 +-----+----------------------+-------+--------+--------+-------+--------+--------+----------+-------+-----+
14:16:28 | P/F |         NAME         |  FLR  |  MEAN  |  CEIL  |  GAP  |  MIN   |  MAX   |   RNG    |  COV  | ITS |
14:16:28 +-----+----------------------+-------+--------+--------+-------+--------+--------+----------+-------+-----+
14:16:28 | P   | boot-times           | 95.0% | 101.0% | 105.0% | 10.0% | -15.0% | 254.7% | -1797.8% | 42.7% |  20 |
14:16:28 | P   | memory-footprint     | 95.0% | 101.0% | 105.0% | 10.0% | 101.0% | 101.0% | 0.0%     | 0.0%  |   1 |
14:16:28 | P   | memory-footprint-ksm | 95.0% | 101.0% | 105.0% | 10.0% | 101.0% | 101.0% | 0.0%     | 0.0%  |   1 |
14:16:28 | P   | blogbench            | 90.0% | 100.8% | 110.0% | 20.0% | 100.8% | 100.8% | 0.0%     | 0.0%  |   1 |
14:16:28 | P   | blogbench            | 90.0% | 98.6%  | 110.0% | 20.0% | 98.6%  | 98.6%  | 0.0%     | 0.0%  |   1 |
14:16:28 +-----+----------------------+-------+--------+--------+-------+--------+--------+----------+-------+-----+
```

**Comment by fidencio** [comment]
> /test-ubuntu-metrics

```
14:48:22 Report Summary:
14:48:22 +-----+----------------------+-------+--------+--------+-------+--------+--------+---------+-------+-----+
14:48:22 | P/F |         NAME         |  FLR  |  MEAN  |  CEIL  |  GAP  |  MIN   |  MAX   |   RNG   |  COV  | ITS |
14:48:22 +-----+----------------------+-------+--------+--------+-------+--------+--------+---------+-------+-----+
14:48:22 | P   | boot-times           | 95.0% | 97.9%  | 105.0% | 10.0% | -35.8% | 258.4% | -821.6% | 48.5% |  20 |
14:48:22 | P   | memory-footprint     | 95.0% | 101.0% | 105.0% | 10.0% | 101.0% | 101.0% | 0.0%    | 0.0%  |   1 |
14:48:22 | P   | memory-footprint-ksm | 95.0% | 101.1% | 105.0% | 10.0% | 101.1% | 101.1% | 0.0%    | 0.0%  |   1 |
14:48:22 | P   | blogbench            | 90.0% | 100.8% | 110.0% | 20.0% | 100.8% | 100.8% | 0.0%    | 0.0%  |   1 |
14:48:22 | P   | blogbench            | 90.0% | 97.9%  | 110.0% | 20.0% | 97.9%  | 97.9%  | 0.0%    | 0.0%  |   1 |
14:48:22 +-----+----------------------+-------+--------+--------+-------+--------+--------+---------+-------+-----+
```

Seems okay-ish enough.

**Review by devimc — APPROVED** [review]
thanks @fidencio - lgtm

cc @GabyCT

**Comment by fidencio** [comment]
Aha, we have the first failure!
```
15:36:07 Report Summary:
15:36:07 +-----+----------------------+-------+--------+--------+-------+--------+--------+---------+-------+-----+
15:36:07 | P/F |         NAME         |  FLR  |  MEAN  |  CEIL  |  GAP  |  MIN   |  MAX   |   RNG   |  COV  | ITS |
15:36:07 +-----+----------------------+-------+--------+--------+-------+--------+--------+---------+-------+-----+
15:36:07 | *F* | boot-times           | 95.0% | 90.6%  | 105.0% | 10.0% | -33.1% | 254.5% | -869.8% | 59.6% |  20 |
15:36:07 | P   | memory-footprint     | 95.0% | 100.9% | 105.0% | 10.0% | 100.9% | 100.9% | 0.0%    | 0.0%  |   1 |
15:36:07 | P   | memory-footprint-ksm | 95.0% | 101.1% | 105.0% | 10.0% | 101.1% | 101.1% | 0.0%    | 0.0%  |   1 |
15:36:07 | P   | blogbench            | 90.0% | 101.5% | 110.0% | 20.0% | 101.5% | 101.5% | 0.0%    | 0.0%  |   1 |
15:36:07 | P   | blogbench            | 90.0% | 98.1%  | 110.0% | 20.0% | 98.1%  | 98.1%  | 0.0%    | 0.0%  |   1 |
15:36:07 +-----+----------------------+-------+--------+--------+-------+--------+--------+---------+-------+-----+
```

**Comment by devimc** [comment]
@fidencio aha, looks like ubuntu image is more stable than fedora, should we revert my change and go back to ubuntu? the big question is from where (registry) we can pull that image?

**Comment by fidencio** [comment]
> @fidencio aha, looks like ubuntu image is more stable than fedora, should we revert my change and go back to ubuntu? the big question is from where (registry) we can pull that image?

I'm not happy about using the ubuntu image again.  I wonder, whether relaxing the limits for Fedora (7.5% min and 5% max) would be okay.  If not, we should give it a try with the CentOS image.

Does this sound reasonable, @devimc?

**Comment by devimc** [comment]
is CentOS Stream more stable than fedora? :thinking:  
can we pull the ubuntu image from here https://gallery.ecr.aws/ubuntu/ubuntu?  do you know what's the rate limit (if any) of ECR?

**Comment by GabyCT** [comment]
any idea why the boot time increase?

File: cmd/checkmetrics/ci_worker/checkmetrics-json-qemu-kata-metric4.toml [hunk 1]
```diff
@@ -16,9 +16,9 @@ description = "measure container lifecycle timings"
 # within (inclusive)
 checkvar = ".\"boot-times\".Results | .[] | .\"to-workload\".Result"
 checktype = "mean"
-midval = 0.575
-minpercent = 10.0
-maxpercent = 10.0
+midval = 0.62
+minpercent = 7.5
+maxpercent = 5.0
```
Comment (devimc) [inline cmd/checkmetrics/ci_worker/checkmetrics-json-qemu-kata-metric4.toml:21]: please change both to 10.0

**Comment by fidencio** [comment]
> is CentOS Stream more stable than fedora? thinking
> can we pull the ubuntu image from here https://gallery.ecr.aws/ubuntu/ubuntu? do you know what's the rate limit (if any) of ECR?

I'd say so, let me try with the CentOS one and we can adjust the limits then.

**Comment by fidencio** [comment]
> any idea why the boot time increase?

Seems that it's happening due to instability on the image used. :-/

**Comment by fidencio** [comment]
> > is CentOS Stream more stable than fedora? thinking
> > can we pull the ubuntu image from here https://gallery.ecr.aws/ubuntu/ubuntu? do you know what's the rate limit (if any) of ECR?
> 
> I'd say so, let me try with the CentOS one and we can adjust the limits then.

Still, about ECR, I don't know how stable it's.  If you think it's worth a try, I'd be game for that chage, as long as we don't get back to docker.io. :-)

**Comment by fidencio** [comment]
First test with ubuntu image:
```
18:26:43 Report Summary:
18:26:43 +-----+--------------------------+---------------------+---------------+--------+-------+--------+--------+-------+-------+-----+
18:26:43 | P/F |           NAME           |         FLR         |     MEAN      |  CEIL  |  GAP  |  MIN   |  MAX   |  RNG  |  COV  | ITS |
18:26:43 +-----+--------------------------+---------------------+---------------+--------+-------+--------+--------+-------+-------+-----+
18:26:43 | P   | boot-times               | 95.0%               | 102.8%        | 105.0% | 10.0% | 95.2%  | 164.5% | 72.9% | 13.9% |  20 |
18:26:43 | P   | memory-footprint         | 95.0%               | 101.0%        | 105.0% | 10.0% | 101.0% | 101.0% | 0.0%  | 0.0%  |   1 |
18:26:43 | P   | memory-footprint-ksm     | 95.0%               | 101.2%        | 105.0% | 10.0% | 101.2% | 101.2% | 0.0%  | 0.0%  |   1 |
18:26:43 | P   | blogbench                | 90.0%               | 97.0%         | 110.0% | 20.0% | 97.0%  | 97.0%  | 0.0%  | 0.0%  |   1 |
18:26:43 | P   | blogbench                | 90.0%               | 96.8%         | 110.0% | 20.0% | 96.8%  | 96.8%  | 0.0%  | 0.0%  |   1 |
18:26:43 | *F* | network-iperf3-bandwidth | Failed to load JSON | exit status 2 |        |       |        |        |       |       |     |
18:26:43 +-----+--------------------------+---------------------+---------------+--------+-------+--------+--------+-------+-------+-----+
```

**Comment by fidencio** [comment]
This is from Julio's run:
```
18:53:24 Report Summary:
18:53:24 +-----+----------------------+-------+--------+--------+-------+--------+--------+-------+-------+-----+
18:53:24 | P/F |         NAME         |  FLR  |  MEAN  |  CEIL  |  GAP  |  MIN   |  MAX   |  RNG  |  COV  | ITS |
18:53:24 +-----+----------------------+-------+--------+--------+-------+--------+--------+-------+-------+-----+
18:53:24 | P   | boot-times           | 95.0% | 101.9% | 105.0% | 10.0% | 95.2%  | 162.1% | 70.3% | 13.7% |  20 |
18:53:24 | P   | memory-footprint     | 95.0% | 101.0% | 105.0% | 10.0% | 101.0% | 101.0% | 0.0%  | 0.0%  |   1 |
18:53:24 | P   | memory-footprint-ksm | 95.0% | 101.1% | 105.0% | 10.0% | 101.1% | 101.1% | 0.0%  | 0.0%  |   1 |
18:53:24 | P   | blogbench            | 90.0% | 99.7%  | 110.0% | 20.0% | 99.7%  | 99.7%  | 0.0%  | 0.0%  |   1 |
18:53:24 | P   | blogbench            | 90.0% | 94.5%  | 110.0% | 20.0% | 94.5%  | 94.5%  | 0.0%  | 0.0%  |   1 |
18:53:24 +-----+----------------------+-------+--------+--------+-------+--------+--------+-------+-------+-----+
```

**Review by GabyCT — APPROVED** [review]

**Comment by GabyCT** [comment]
It passed the CI...let me run again to see that it is stable

**Comment by fidencio** [comment]
Thanks, @GabyCT!

**Comment by fidencio** [comment]
```
19:22:58 Report Summary:
19:22:58 +-----+----------------------+-------+--------+--------+-------+--------+--------+-------+-------+-----+
19:22:58 | P/F |         NAME         |  FLR  |  MEAN  |  CEIL  |  GAP  |  MIN   |  MAX   |  RNG  |  COV  | ITS |
19:22:58 +-----+----------------------+-------+--------+--------+-------+--------+--------+-------+-------+-----+
19:22:58 | P   | boot-times           | 95.0% | 101.4% | 105.0% | 10.0% | 93.9%  | 160.5% | 71.0% | 13.6% |  20 |
19:22:58 | P   | memory-footprint     | 95.0% | 100.9% | 105.0% | 10.0% | 100.9% | 100.9% | 0.0%  | 0.0%  |   1 |
19:22:58 | P   | memory-footprint-ksm | 95.0% | 101.0% | 105.0% | 10.0% | 101.0% | 101.0% | 0.0%  | 0.0%  |   1 |
19:22:58 | P   | blogbench            | 90.0% | 101.3% | 110.0% | 20.0% | 101.3% | 101.3% | 0.0%  | 0.0%  |   1 |
19:22:58 | P   | blogbench            | 90.0% | 98.8%  | 110.0% | 20.0% | 98.8%  | 98.8%  | 0.0%  | 0.0%  |   1 |
19:22:58 +-----+----------------------+-------+--------+--------+-------+--------+--------+-------+-------+-----+
```

**Comment by devimc** [comment]
```
13:33:55 +-----+----------------------+-------+--------+--------+-------+--------+--------+-------+-------+-----+
13:33:55 | P/F |         NAME         |  FLR  |  MEAN  |  CEIL  |  GAP  |  MIN   |  MAX   |  RNG  |  COV  | ITS |
13:33:55 +-----+----------------------+-------+--------+--------+-------+--------+--------+-------+-------+-----+
13:33:55 | P   | boot-times           | 95.0% | 100.5% | 105.0% | 10.0% | 93.2%  | 162.7% | 74.6% | 14.5% |  20 |
13:33:55 | P   | memory-footprint     | 95.0% | 101.0% | 105.0% | 10.0% | 101.0% | 101.0% | 0.0%  | 0.0%  |   1 |
13:33:55 | P   | memory-footprint-ksm | 95.0% | 101.2% | 105.0% | 10.0% | 101.2% | 101.2% | 0.0%  | 0.0%  |   1 |
13:33:55 | P   | blogbench            | 90.0% | 100.2% | 110.0% | 20.0% | 100.2% | 100.2% | 0.0%  | 0.0%  |   1 |
13:33:55 | P   | blogbench            | 90.0% | 98.7%  | 110.0% | 20.0% | 98.7%  | 98.7%  | 0.0%  | 0.0%  |   1 |
13:33:55 +-----+----------------------+-------+--------+--------+-------+--------+--------+-------+-------+-----+
```

**Comment by devimc** [comment]
all green again

```
14:05:52 +-----+----------------------+-------+--------+--------+-------+--------+--------+-------+-------+-----+
14:05:52 | P/F |         NAME         |  FLR  |  MEAN  |  CEIL  |  GAP  |  MIN   |  MAX   |  RNG  |  COV  | ITS |
14:05:52 +-----+----------------------+-------+--------+--------+-------+--------+--------+-------+-------+-----+
14:05:52 | P   | boot-times           | 95.0% | 101.9% | 105.0% | 10.0% | 92.9%  | 158.2% | 70.3% | 13.0% |  20 |
14:05:52 | P   | memory-footprint     | 95.0% | 101.0% | 105.0% | 10.0% | 101.0% | 101.0% | 0.0%  | 0.0%  |   1 |
14:05:52 | P   | memory-footprint-ksm | 95.0% | 101.1% | 105.0% | 10.0% | 101.1% | 101.1% | 0.0%  | 0.0%  |   1 |
14:05:52 | P   | blogbench            | 90.0% | 95.9%  | 110.0% | 20.0% | 95.9%  | 95.9%  | 0.0%  | 0.0%  |   1 |
14:05:52 | P   | blogbench            | 90.0% | 93.8%  | 110.0% | 20.0% | 93.8%  | 93.8%  | 0.0%  | 0.0%  |   1 |
14:05:52 +-----+----------------------+-------+--------+--------+-------+--------+--------+-------+-------+-----+
```

**Comment by fidencio** [comment]
Let's run all the tests (including the metrics again) and have it merged.

**Comment by GabyCT** [comment]
Merging this PR , the ARM failure is well known and it already has an issue

**Pull request merged**